site/src/atomics/flex.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Flex Atomics
+description: Flex related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Flex Atomics
+
+Flex atomics are useful when composing layouts and patterns containing other components.
+
+| cssproperty       | value                                                                               | screensize |
+| ----------------- | ----------------------------------------------------------------------------------- | ---------- |
+| `flex-direction`  | `row`, `column`, `row-reverse`, `column-reverse`                                    | `tablet`   |
+| `justify-content` | `flex-start`, `flex-end`, `center`, `space-around`, `space-between`, `space-evenly` | `tablet`   |
+| `align-items`     | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `align-self`      | `flex-start`, `flex-end`, `center`, `baseline`, `stretch`                           | `tablet`   |
+| `flex-wrap`       | `nowrap`, `wrap`, `wrap-reverse`                                                    | `tablet`   |
+| `align-content`   | `flex-start`, `flex-end`, `space-around`, `space-between`, `stretch`                | `tablet`   |
+| `flex-grow`       | `0`, `1`                                                                            | `tablet`   |
+| `flex-shrink`     | `0`, `1`                                                                            | `tablet`   |
+
+## Usage
+
+The following example uses a combination of mobile-first flex atomics to achieve a responsive card layout.
+
+```html
+<article
+	id="example-layout"
```

Comment:
Should be better now 
![image](https://user-images.githubusercontent.com/94572161/148120461-a1c8fc95-2a27-4591-baf9-7c20944b6735.png)

Reply:
Still looks quite odd to me. Given the description I thought we were going for the following:

`display-flex flex-direction-column flex-direction-row-tablet`

Then add some padding to the blue container like Olga suggested and it may be fine.

Reply:
![image](https://user-images.githubusercontent.com/30843002/148121106-9aca4ce0-7c85-4fa4-88cc-228693e8c216.png)

Reply:
![image](https://user-images.githubusercontent.com/30843002/148121128-bc959e40-4951-4d49-b70f-a4c473e92364.png)

Reply:
I used `padding-md`

Reply:
Updated